### ISSUE TITLE
App doesn't show Dutch texts when language set to Dutch

### ISSUE THREAD

**Issue by Joennuh** [VREMSoftwareDevelopment/WiFiAnalyzer#259]
**Please search issues (open and closed) to see the issues has already been reported.**
Only found https://github.com/VREMSoftwareDevelopment/WiFiAnalyzer/issues/29 ("Dutch Translations") which isn't actually a bug. Since the last activity was on July 25th, 2016 I don't know whether that topic is relevant anymore.

**Describe the bug**
I've set the language to "Nederlands (Nederland)" (in English this would be "Dutch (Netherlands)") but the app keeps on being displayed in English. My Android language is set to Dutch.

**To Reproduce**
1. Go to 'Settings'.
2. Tap 'Language'.
3. Choose 'Nederlands (Nederland)'.
4. Close the app.
5. Re-open the app.
6. The app stays in English.

**Expected behavior**
When the app language is set to Nederlands (Dutch) it should display all texts in Dutch. Elsewhise the language options shouldn't be present if there is no translation available.

**Screenshots**
Not applicable. The description speaks for its self.

**Note**
If needed I'm willing to help to translate into Dutch. The only thing is that I haven't any experience with programming Android apps nor using GIT. But when a list of string to translate can be provided by mail then I will be able to translate them and set it back to you.

**Additional context**
My English is fine as you can see but reading all texts in Dutch in the app would be pleasant.

**Provide Device/Android OS information**
HMD Nokia 8, model TA-1012, Android 9, securitypatch August 1st, 2019

**Comment by VREMSoftwareDevelopment** [comment]
You are correct this is not bug.
Dutch translation was never done.
#29 was closed, since no-one provided the translation.
If you are willing to provide Dutch translation, new contributors are always welcome to the project.

Translation References in Code:

- [English](https://github.com/VREMSoftwareDevelopment/WiFiAnalyzer/blob/master/app/src/main/res/values/strings.xml)
- [French](https://github.com/VREMSoftwareDevelopment/WiFiAnalyzer/blob/master/app/src/main/res/values-fr/strings.xml)
- [German](https://github.com/VREMSoftwareDevelopment/WiFiAnalyzer/blob/master/app/src/main/res/values-de/strings.xml)

[How to submit a pull request](https://github.com/VREMSoftwareDevelopment/WiFiAnalyzer/wiki/Pull-Request)

**Comment by Joennuh** [comment]
I'm willing to translate the texts into Dutch for you but I'm not familiar enough with GIT and all its functionalities to be able to add the file to the repository by my self. For me it's best to email my translated file to someone who is more familiar with GIT. Otherwise I'll need to to do a course or something like that and that will take some time.

**Comment by VREMSoftwareDevelopment** [comment]
Please find attached English file:
[strings.zip](https://github.com/VREMSoftwareDevelopment/WiFiAnalyzer/files/3586012/strings.zip)

Please translate entries that do not have `translatable="false"`

**Comment by Joennuh** [comment]
I've tried it the official way but I found it a bit difficult. I hope I did all in the right way.
At first I forgot to checkout on the feature branch.
Then I discovered that I can not commit to the WiFiAnalyzer respository. I didn't have any knowledge about the "Pull request" system yet and so I had to familiarize my self with it and so I dscovered I had to fork the project, place my translations on my forked respository, commit the changes to the forked respository and then place the pull request with the correct text to close the issue.
That is quite a big learning curve for someone who didn't use Git and Github much on this way. 😛
